web/templates/web/studies-list.html

```diff
@@ -4,96 +4,125 @@
-                                            {% endif %}
-                                          </div>
-                                          <div class="caption">
-                                              <h3>{{obj.name}}</h3>
-                                              <p><em>{{obj.criteria}}</em></p>
-                                              <p>{{obj.short_description}}</p>
-                                              <div class="fadeout"></div>
-                                          </div>
-                                          <div class="full">
-                                              <p class="text-center"><a href="{% url 'web:study-detail' uuid=obj.uuid %}">{% trans "See details" %}</a></p>
-                                          </div>
-                                      </a>
-                                  </div>
-                                </div>
-                              {% empty %}
-                                <div class="col-xs-12">
-                                    <p><em> {% trans "We are not running any studies at this time!" %} </em></p>
-                                </div>
-                              {% endfor %}
-                        </div>
-                    </div>
-                    <div class="col-xs-12">
-                        <p class="study-detail-caption">{% trans 'Looking for more ways to contribute to research from home? Check out <a href="https://childrenhelpingscience.com/" target="_blank" rel="noreferrer noopener">Children Helping Science</a> for even more studies!' %}</p>
+</div>
+
+<div class="panel-default">
+    <div class="panel-body">
+        {% trans "Please note you'll need a laptop or desktop computer (not a mobile device) running Chrome or Firefox to participate." %}
+    </div>
+</div>
+
+{% for study in object_list %}
+<div class="panel panel-default study-list">
+    <div class="panel-body">
+        <div class="row">
+            <div class="col-md-3 left">
+                <img src="{{study.image.url}}" class="img-responsive img-rounded" alt="Responsive image">
+                <div class="study-criteria">
+                    <small>
+                        <p>{{study.criteria}}</p>
+                        {% if study.eligible_children %}
+                        <p>{{ study.eligible_children }} maybe eligible for this study.</p>
+                        {% endif %}
+                        </small>
+                    <div class="text-center">
+                        <a class="btn {% if study.eligible_children %}btn-success{% endif %} btn-default text-center" href="{% url 'web:study-detail' uuid=study.uuid %}" role="button">
+                            {% trans "See details" %}
+                        </a>
                     </div>
                 </div>
             </div>
+            <div class="col-md-9 right">
+                <h1>{{study.name}}</h1>
+                <h2>{{study.lab.name}}</h2>
```

Comment:
do we have space here to add (Institution)? A lot of lab names are very similar and the institution is an easier way to distinguish